website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,325 @@
+<template lang="pug">
+  b-modal#guild-form(:title="title", :hide-footer="true")
+    form(@submit.stop.prevent="submit")
+      .form-group
+        label
+          strong(v-once) {{$t('name')}}*
+        b-form-input(type="text", placeholder="$t('newGuildPlaceHolder')", v-model="newGuild.name")
+
+      .form-group(v-if='newGuild.id')
+        label
+          strong(v-once) {{$t('guildLeader')}}*
+          b-form-select(v-model="newGuild.newLeader" :options="members")
+
+      .form-group
+        label
+          strong(v-once) {{$t('privacySettings')}}*
+        br
+        b-form-checkbox(v-model='newGuild.onlyLeaderCreatesChallenges') {{$t('onlyLeaderCreatesChallenges')}}
+          b-tooltip.icon(:content="$t('privateDescription')")
+            img(src='~assets/guilds/information.svg')
+        br
+        b-form-checkbox(v-model='newGuild.guildLeaderCantBeMessaged') {{$t('guildLeaderCantBeMessaged')}}
```

Comment:
let's use normal checkboxes in place of the components' ones and also in place of `b-form-input`